Makefile

```diff
@@ -338,9 +338,9 @@ smoketest:
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=p1am-100            examples/blinky1
 	@$(MD5SUM) test.hex
-	$(TINYGO) build -size short -o test.hex -target=atsame54-xpro       examples/blinky1
+	$(TINYGO) build -size short -o test.hex -target=atsame54-xpro       examples/can
```

Comment:
Should probably build both `blinky1` and also `can` in the smoke test, since they are testing different things, right?